go/src/koding/socialapi/transport_test.go

```diff
@@ -0,0 +1,142 @@
+package socialapi_test
+
+import (
+	"encoding/json"
+	"log"
+	"net"
+	"net/http"
+	"strings"
+	"testing"
+
+	"koding/kites/tunnelproxy/discover/discovertest"
+	"koding/socialapi"
+)
+
+func TestTransport(t *testing.T) {
+	var storage TrxStorage
```

Comment:
No, I can't. The TrxStorage is kind of recording storage. It records all calls to Set / Get / Delete functions and then we verify in tests.